files/initramfs-tools/union-mount.j2

```diff
@@ -39,10 +39,38 @@ set_tmpfs_log_partition_size()
   [ $maxsize -le $varlogsize ] && varlogsize=$maxsize
 }
 
+whitelist_rw_folder()
+{
+  image_dir=$1
+  whitelist_file=${rootmnt}/host/$image_dir/whitelist_paths.conf
+  
+  # Return if the whitelist file does not exist
+  if ! test -f "${whitelist_file}"; then
```

Comment:
Should this check be moved down or add as a protection of accessing the file?

If secure boot is enabled and this file is missing, should you at least whitelist rw_dir? I understand that is not enough.

Reply:
The file should not be missing, it is extracted from the image every time in boot0.
If secure boot enabled, the file should always exist, I will improve it.